app/lib/admin/metrics/measure/instance_media_attachments_measure.rb

```diff
@@ -50,7 +50,7 @@ def sql_query_string
           WHERE date_trunc('day', media_attachments.created_at)::date = axis.period
             AND #{account_domain_sql(params[:include_subdomains])}
         )
-        SELECT COALESCE(SUM(size), 0) FROM new_media_attachments
+        SELECT COALESCE(SUM(size), 0)::int FROM new_media_attachments
```

Comment:
Looking more into it, `SUM(size)` used to return a `bigint` because `size` was an `int`, but now that `size` is a `bigint`, `SUM(size)` is a `numeric`, which is handled as `float` in Ruby.

Casting to `int` will cause `PG::NumericValueOutOfRange: ERROR:  integer out of range` if the cumulated size reaches 2GB.

```suggestion
        SELECT COALESCE(SUM(size), 0)::bigint FROM new_media_attachments
```